dashboard/app/models/script_level.rb

```diff
@@ -675,7 +675,23 @@ def get_example_solutions(level, current_user, section_id=nil)
 
     return [] if !Policies::InlineAnswer.visible_for_script_level?(current_user, self) || CDO.properties_encryption_key.blank?
 
-    if level.try(:examples).present? && (current_user&.verified_instructor? || script&.csf?) # 'solutions' for applab-type levels
+    # exemplar_sources is used by Javalab levels to store level solutions
+    if level.try(:exemplar_sources).present? && current_user&.verified_instructor?
+      if oldest_active_level.is_a? BubbleChoice
+        # If the script level has sublevels, get a link for the sublevel that looks like
+        # /csa1/lessons/6/levels/5/sublevel/1?exemplar=true
+        sublevel_position = oldest_active_level.sublevel_position(level)
+        return [] unless sublevel_position
+
+        path = build_script_level_path(self, {sublevel_position: sublevel_position})
+        level_example_links = [build_exemplar_url(path)]
+      else
+        # Otherwise, exemplar link should look like
+        # csa1/lessons/2/levels/1?exemplar=true
+        path = build_script_level_path(self)
+        level_example_links = [build_exemplar_url(path)]
```

Comment:
The URLs generated here are always for production, which is consistent with all of the other `example_solution` links generated in this method. Felt like it was better to be consistent.

Reply:
I think the reason we use prod for the other links is because they other links are all projects which are tied to an environment. We don't need to follow that pattern here right? I think it makes sense to deviate from that pattern here and use the actual environment.

Reply:
mmm good point, didn't think of that. I'll update